eggfruit.lua

```diff
@@ -0,0 +1,171 @@
+
+--[[
+
+Copyright (C) 2015 - Auke Kok <sofar@foo-projects.org>
+              2018 - Marius Spix <marius.spix@web.de>
```

Comment:
This is fine, but, you didn't claim or license the PNG files in the LICENSE file. This is required - make sure to document the origin of all the textures and choose an appropriate license (if it's the same license as the other images, perfect. If it's not, we can discuss)